config/routes.rb

```diff
@@ -3,4 +3,5 @@
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   root to: 'organizations#index'
   resources :projects, only: [:index]
+  get 'projects/:organization_id/dashboard', to: 'projects#simple'
```

Comment:
The other view is more "simple!" We should just call this "dashboard" consistently. (This can wait on the other path changes I mentioned.)